src/main/webapp/WEB-INF/tags/instructor/copyFromModal.tag

```diff
@@ -0,0 +1,90 @@
+<%@ tag description="instructorFeedbacks  - Copy From Another FS modal" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+
+<%@ attribute name="fsList" type="teammates.ui.template.FeedbackSessionsList" required="true"%>
+<%@ attribute name="newForm" type="teammates.ui.template.FeedbackSessionsNewForm" required="true"%>
+
```

Comment:
If you only require the feedback sessions from `fsList`, why not just pass that?
Same goes for `newForm`.